content/salary-report/2020-02/index.ru.md

```diff
@@ -506,58 +506,58 @@ date: 2020-03-05T16:19:24+05:00
 
 <!-- prettier-ignore -->
 - I filled up the form based on my job experience which I quitted recently because they refused to pay me the amount I requested.They underestimate QA Engineers and testers in most software companies in Uzbekistan which is why the quality of software products are low.
-- На сколько изменилась зарплата - не понятно как поставить ответ на 10%
```

Comment:
No need to translate the user feedback because they were left from users. In Uzbek version we also leave them as is.